### PR TITLE
Support special unicode like Chinese and Japanese

### PR DESCRIPTION
**Pull request by LingyuCoder**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,13 +1,14 @@
 {
   "name": "cli-table2",
-  "version": "0.1.9",
+  "version": "0.1.10",
```
Comment (jamestalmage) [inline package.json]: Don't bump the version. That is handled automatically when I cut a release.

File: src/utils.js [hunk 2]
```diff
@@ -1,16 +1,10 @@
 var _ = require('lodash');
+var strlen = require('widest-line');
```
Comment (jamestalmage) [inline src/utils.js]: var widestLine = require('widest-line');

File: test/utils-test.js [hunk 3]
```diff
@@ -357,4 +365,4 @@ describe('utils',function(){
     });
 
   });
-});
\ No newline at end of file
+});
```
Comment (jamestalmage) [inline test/utils-test.js]: Don't make unrelated changes.

**Comment by jamestalmage** [comment]
Looks good to me, just a few comments.

If you can address those and make and amended commit, that would be great.

**Comment by jamestalmage** [comment]
Oh shoot. @LingyuCoder - I didn't see that you updated with my requested changes. Thanks. I will review again and merge soon.

File: package.json [hunk 4]
```diff
@@ -7,7 +7,8 @@
     "test": "test"
   },
   "dependencies": {
-    "lodash": "^3.10.1"
+    "lodash": "^3.10.1",
+    "widest-line": "~1.0.0"
```
Comment (sindresorhus) [inline package.json]: Should use `^` version range.

File: src/utils.js [hunk 5]
```diff
@@ -1,16 +1,11 @@
 var _ = require('lodash');
+var widestLine = require('widest-line');
+var strlen = widestLine;
```
Comment (sindresorhus) [inline src/utils.js]: I would just use the original name `widestLine` which is clearer.

**Comment by jamestalmage** [comment]
@LingyuCoder - Thanks for this. I ended up going with #17, as @fand took that PR a lot further. You got the ball rolling though, so thanks for that.

Proper CJK character support deployed in `0.2.0`
